sources.go

```diff
@@ -113,3 +113,19 @@ func Slice(s []string) *Pipe {
 func Stdin() *Pipe {
 	return NewPipe().WithReader(os.Stdin)
 }
+
+// Prompt presents a message to the user and returns a pipe containing user input or defaultValue provided.
+func Prompt(message, defaultValue string) *Pipe {
```

Comment:
To make it easier to find things, functions are listed in alphabetical order in the source files (see the [Contributing](https://github.com/bitfield/script/blob/master/CONTRIBUTING.md) guide for details).

Reply:
Corrected